modules/reactor/test/tests/meshgenerators/polygon_concentric_circle_mesh_generator/tests

```diff
@@ -141,6 +141,47 @@
     requirement ='The system shall generate a hexagon mesh with rings, ducts and quad central elements with custom center quad factor.'
     recover = false
   []
+  [reporting_id]
+    requirement = 'The system shall generate a pin mesh with the reporting ID'
+    [annular_block]
+      type = 'Exodiff'
+      input = 'pin_reporting_id.i'
+      exodiff = 'pin_reporting_id_annular_block_out.e'
+      cli_args = 'Mesh/pin/use_ring_id=true
+                  Mesh/pin/ring_id_name="reporting_id"
```

Comment:
what are you thoughts on just having a ring_id_name parameter, and implicitly set use_ring_id

Reply:
I think that would be a better idea. I will update as you suggested.